shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/AboutConference.kt

```diff
@@ -1,21 +1,211 @@
+                location = "Hall A",
+                time = "10:00 – 11:00",
+            )
+
+            Event(
+                month = "MAY",
+                day = "22",
+                day2 = "–23",
+                line1 = stringResource(Res.string.about_conference_codelabs_title_line1),
+                line2 = stringResource(Res.string.about_conference_codelabs_title_line2),
+                description = "TBA",
+                speakers = emptyList(),
+                location = "",
+                time = "",
+                backgroundColor = KotlinConfTheme.colors.tileBackground,
+            )
+
+            Event(
+                month = "MAY",
+                day = "22",
+                line1 = stringResource(Res.string.about_conference_party_line1),
+                line2 = stringResource(Res.string.about_conference_party_line2),
+                description = stringResource(Res.string.about_conference_party_description),
+                speakers = emptyList(),
+                location = "Hall A",
+                time = "18:00 – 22:30",
+                backgroundColor = KotlinConfTheme.colors.tileBackground,
+            )
+
+            Event(
+                month = "MAY",
+                day = "23",
+                line1 = stringResource(Res.string.about_conference_closingpanel_title_line1),
+                line2 = stringResource(Res.string.about_conference_closingpanel_title_line2),
+                description = "",
+                speakers = listOf(
+                    Speaker(SpeakerId("3"), "Hadi Hariri", "", "Never believed in elevator pitches", ""),
+                ),
+                location = "Hall A",
+                time = "17:00 – 18:00",
+            )
+        }
+
+        Image(
+            painter = painterResource(Res.drawable.kotlinconf_by_jetbrains),
+            contentDescription = stringResource(Res.string.kotlinconf_by_jetbrains_description),
+            modifier = Modifier.align(Alignment.CenterHorizontally).padding(vertical = 64.dp)
+        )
+
+        Column(
+            modifier = Modifier.padding(bottom = 24.dp),
+            verticalArrangement = Arrangement.spacedBy(12.dp),
+        ) {
+            PageMenuItem(stringResource(Res.string.about_conference_privacy_policy_link), onClick = onPrivacyPolicy)
+            PageMenuItem(stringResource(Res.string.about_conference_general_terms_link), onClick = onGeneralTerms)
+            PageMenuItem(stringResource(Res.string.about_conference_website_link), drawableResource = Res.drawable.arrow_up_right_24, onClick = onWebsiteLink)
+        }
+    }
+}
+
+@Composable
+private fun Event(
+    month: String,
+    day: String,
+    line1: String,
+    line2: String,
+    description: String,
+    speakers: List<Speaker>,
+    location: String,
+    time: String,
+    day2: String = "",
+    backgroundColor: Color = KotlinConfTheme.colors.mainBackground,
+) {
+    val roundedCornerShape = RoundedCornerShape(8.dp)
+    Column(
```

Comment:
Some of the vertical padding/spacing here is too much compared to the design. In both of these examples it should be less:

<img width="592" alt="image" src="https://github.com/user-attachments/assets/d5ea6f93-e72f-4a57-b62c-854b55b747a9" />
<img width="594" alt="image" src="https://github.com/user-attachments/assets/69c0156d-8b14-4442-97d5-4f594a38da9d" />